src/Illuminate/Support/Facades/Request.php

```diff
@@ -62,7 +62,7 @@
  * @method static string userAgent()
  * @method static \Illuminate\Http\Request merge(array $input)
  * @method static \Illuminate\Http\Request replace(array $input)
- * @method static \Symfony\Component\HttpFoundation\ParameterBag|mixed json(string|null $key = null, mixed $default = null)
```

Comment:
Please revert this change.

Reply:
I changed other facades as well. You want them back as well?

Reply:
Any return types that used to have an object combined with mixed, should be retained.

Reply:
Reverted all **return types** where object was combined with mixed.